src/cookie.ts

```diff
@@ -1,4 +1,4 @@
-import cookie from 'cookie';
+import * as cookie from 'cookie';
```

Comment:
can we import only what is needed?

Reply:
Make sense, I will make changes in next commit